src/backend/tools/index.ts

```diff
@@ -830,3 +834,183 @@ export async function runWineCommandOnGame(
     startFolder
   })
 }
+
+const STEAM_DOWNLOAD_URL = 'https://archive.org/download/SteamHeroic/Steam.zip'
+
+/**
+ * Download and Install the Windows version of Steam on a Wine Prefix on macOS only
+ */
+export const SteamWindows = {
+  downloadSteam: async () => {
+    const directory = `${toolsPath}/steam`
+    const fileName = 'Steam.zip'
+    const window = getMainWindow()
+
+    if (!existsSync(directory)) {
+      mkdirSync(directory, { recursive: true })
+    }
+
+    if (!isOnline() || existsSync(join(directory, fileName)) || !window) {
+      return
+    }
+
+    const abortController = createAbortController('steam-download')
+
+    function handleProgress(
+      downloadedBytes: number,
+      downloadSpeed: number,
+      progress: number
+    ) {
+      window?.webContents.send(`progressUpdate-steam`, {
+        appName: 'steam',
+        status: 'installing',
+        runner: 'sideload',
+        progress: {
+          percent: progress,
+          bytes: '718394478'
+        }
+      })
+    }
+
+    try {
+      await downloadFile({
+        url: STEAM_DOWNLOAD_URL,
+        dest: join(directory, fileName),
+        abortSignal: abortController.signal,
+        progressCallback: handleProgress,
+        ignoreFailure: false
+      })
+    } catch (error) {
+      logError(['Error downloading Steam', error], LogPrefix.Backend)
+      throw new Error('Error downloading Steam')
+    }
+  },
+  installSteam: async (path: string) => {
+    const steamCoverArt =
+      'https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/a7e8ba67562ea4d4ca0421066466ece4.png'
+    const steamSetupPath = `${toolsPath}/steam/Steam.zip`
```

Comment:
This path is defined multiple times (once in the above function, once in this, once again further down). Ideally keep it as one variable in the top level